lib/Utils/Paths.cpp

```diff
@@ -0,0 +1,229 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
```

Comment:
Same question does this person know his code is donated? I'd like to have an internal discussion on this matter. For now, could you remove the authors. This way, the code will align better under our current "contribution" section. We can re-add them later when we resolve this issue. Otherwise, the PR will have to wait for the outcome of the discussion, which can be very long.